src/components/RequestGroupComposition/CustomModal.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <b-modal ref="my-modal" size="xl" :title="header">
+  <b-modal ref="my-modal" size="xl" :title="header" @close="close" @hidden="close">
```

Comment:
This was to fix when you click on the `X` at the top right of the modal, or click on the background of the modal to close it. Previously doing either of those things prevents the modal from being opened again.